package.json

```diff
@@ -20,6 +20,9 @@
   ],
   "license": "MIT",
   "repository": "git@github.com:CatchRelease/arbor.git",
+  "engines": {
+    "node": "~8.11"
```

Comment:
Not required as part of this PR but it would be nice to test the Arbor repo against multiple versions of NodeJS since it's open source. That might require us to move over to something like TravisCI though. I haven't found anything in the Codeship docs about testing against multiple versions.